### PR TITLE
feat(draw): add draw thread stack size config option

### PR DESCRIPTION
**Pull request by lion2tomato**
flexible set of draw thread stacksize,default value set to 32k to fit freetype

### CODE REVIEW

File: src/lv_conf_internal.h [hunk 1]
```diff
@@ -370,6 +370,15 @@
     #endif
 #endif
 
+/* flexible set draw thread stacksize. */
+#ifndef LV_DRAW_THREAD_STACKSIZE
+    #ifdef CONFIG_LV_DRAW_THREAD_STACKSIZE
+        #define LV_DRAW_THREAD_STACKSIZE CONFIG_LV_DRAW_THREAD_STACKSIZE
+    #else
+        #define LV_DRAW_THREAD_STACKSIZE 4096
```
Comment (FASTSHIFT) [inline src/lv_conf_internal.h]: 4096 -> 32768

Comment (lion2tomato) [reply src/lv_conf_internal.h]: done

File: Kconfig [hunk 2]
```diff
@@ -197,6 +197,10 @@ menu "LVGL configuration"
 				> 1 requires an operating system enabled in `LV_USE_OS`
 				> 1 means multiply threads will render the screen in parallel
 
+		config LV_DRAW_THREAD_STACKSIZE
+			int "stack size of draw thread"
+			default 32768
```
Comment (FASTSHIFT) [inline Kconfig:202]: config LV_DRAW_THREAD_STACKSIZE
			int "stack size of draw thread"
			default 32768
                        depends on LV_USE_OS > 0

**Review by FASTSHIFT — CHANGES_REQUESTED** [review]

File: Kconfig (hunk 2 above)
Comment (lion2tomato) [reply Kconfig:202]: done

**Review by FASTSHIFT — DISMISSED** [review]

File: tests/src/lv_test_conf_full.h [hunk 3]
```diff
@@ -1,5 +1,6 @@
 #define LV_MEM_SIZE                     (32 * 1024 * 1024)
 #define LV_DRAW_SW_SHADOW_CACHE_SIZE    8
+#define LV_DRAW_THREAD_STACKSIZE    (128 * 1024)
```
Comment (XuNeo) [inline tests/src/lv_test_conf_full.h]: Let's use the recommended 32kB in default config instead.

File: Kconfig [hunk 4]
```diff
@@ -197,6 +197,11 @@ menu "LVGL configuration"
 				> 1 requires an operating system enabled in `LV_USE_OS`
 				> 1 means multiply threads will render the screen in parallel
 
+		config LV_DRAW_THREAD_STACKSIZE
+			int "stack size of draw thread"
```
Comment (XuNeo) [inline Kconfig]: LVGL has mix use of size unit, let's mention it's byte here.

```suggestion
			int "stack size of draw thread in byte"
```

File: lv_conf_template.h [hunk 5]
```diff
@@ -144,6 +144,9 @@
     #endif
 #endif
 
+/* Set stacksize of drawing thread. */
```
Comment (XuNeo) [inline lv_conf_template.h]:
```suggestion
/* Set stack size of drawing thread. Unit is byte.*/
```

File: tests/src/lv_test_conf_full.h (hunk 3 above)
Comment (lion2tomato) [reply tests/src/lv_test_conf_full.h:3]: Using a 32kb stack will cause a crash, as Thorvg consumes a significant amount of stack memory.

Comment (lion2tomato) [reply tests/src/lv_test_conf_full.h]: Using a 32kb stack will cause stack-overflow, as Thorvg consumes a significant amount of stack memory

**Review by FASTSHIFT — DISMISSED** [review]

File: tests/src/lv_test_conf_full.h (hunk 3 above)
Comment (XuNeo) [inline tests/src/lv_test_conf_full.h]:
```suggestion
#define LV_DRAW_THREAD_STACKSIZE    (128 * 1024) /*Increase stack size to 128kB in order to run Thorvg*/
```

File: lv_conf_template.h [hunk 6]
```diff
@@ -144,6 +144,9 @@
     #endif
 #endif
 
+/* Set stack size of drawing thread. Unit is byte.*/
```
Comment (XuNeo) [inline lv_conf_template.h]:
```suggestion
/* Set stack size of drawing thread. Unit is byte. If Thorvg is enabled, 128kB is required tested on simulator.*/
```

**Review by XuNeo — DISMISSED** [review]

**Review by FASTSHIFT — APPROVED** [review]

**Review by XuNeo — APPROVED** [review]

**Review by PGNetHun — APPROVED** [review]

**Pull request merged**